drivers/soundwire/intel.c

```diff
@@ -1591,12 +1591,17 @@ static int intel_resume_runtime(struct device *dev)
 	return _resume(dev, clock_stop);
 }
 
-#endif
-
 static const struct dev_pm_ops intel_pm = {
 	SET_SYSTEM_SLEEP_PM_OPS(intel_suspend, intel_resume)
 	SET_RUNTIME_PM_OPS(intel_suspend_runtime, intel_resume_runtime, NULL)
 };
+#else
+static const struct dev_pm_ops intel_pm = {
+	SET_SYSTEM_SLEEP_PM_OPS(NULL, NULL)
+	SET_RUNTIME_PM_OPS(NULL, NULL, NULL)
+};
+
+#endif
```

Comment:
no. you can use fallback functions for intel_suspend/resume, etc which return 0 when PM runtime is not needed.